src/index/ivf/ivf.cc

```diff
@@ -231,7 +231,9 @@ IvfIndexNode<T>::Train(const DataSet& dataset, const Config& cfg) {
     }
     // do normalize for COSINE metric type
     if (IsMetricType(base_cfg.metric_type.value(), knowhere::metric::COSINE)) {
-        Normalize(dataset);
+        if constexpr (!(std::is_same_v<faiss::IndexIVFFlatCC, T>)) {
```

Comment:
why use "std::is_same_v"? should be "std::is_same" ?

Reply:
These two expressions are equivalent.
template <typename _Tp, typename _Up>
  inline constexpr bool is_same_v = is_same<_Tp, _Up>::value;

Reply:
since in knowhere we only use "std::is_same", you can see line 67 and line 70,
please keep the behavior sync